packages/components/src/components/mobile-drawer/mobile-drawer-submenu-section.jsx

```diff
@@ -4,25 +4,24 @@ import React from 'react';
 import Icon from '../icon/icon.jsx';
 import Text from '../text/text.jsx';
 
-class SubMenuSection extends React.PureComponent {
```

Comment:
Since this is a pure component. I guess we have to memorize the function component.

we can use React.memo in export

```
export default React.Memo(SubMenuSection);
```

Reply:
agree with you, thx